src/app/screens/Receive/index.tsx

```diff
@@ -41,19 +41,23 @@ function Receive() {
     description: "",
     expiration: "",
   });
-  const [loading, setLoading] = useState(false);
+  const [loadingInvoice, setLoadingInvoice] = useState(false);
   const [invoice, setInvoice] = useState<{
     paymentRequest: string;
     rHash: string;
   } | null>();
-  const [copyLabel, setCopyLabel] = useState(
+  const [copyInvoiceLabel, setCopyInvoiceLabel] = useState(
     tCommon("actions.copy_invoice") as string
   );
+  const [copyLightningAddressLabel, setCopyLightningAddressLabel] = useState(
+    t("actions.copy_lightning_address") as string
+  );
+  const [lightningAddress, setLightningAddress] = useState("");
   const [paid, setPaid] = useState(false);
   const [pollingForPayment, setPollingForPayment] = useState(false);
   const mounted = useRef(false);
   const isAlbyUser =
-    isAlbyOAuthAccount(auth.account?.alias) ||
+    isAlbyOAuthAccount(auth.account?.connectorType) ||
```

Comment:
This is already fixed in master, why is it not in this diff?

Reply:
probably cause pr was created before lnurl-redeem pr was merged!  merged master into the branch, resolved!